Lib/fontTools/feaLib/builder.py

```diff
@@ -1471,7 +1471,9 @@ def add_class_pair_pos(self, location, glyphclass1, value1, glyphclass2, value2)
         lookup = self.get_lookup_(location, PairPosBuilder)
         v1 = self.makeOpenTypeValueRecord(location, value1, pairPosContext=True)
         v2 = self.makeOpenTypeValueRecord(location, value2, pairPosContext=True)
-        lookup.addClassPair(location, glyphclass1, v1, glyphclass2, v2)
+        cls1 = tuple(sorted(set(glyphclass1)))
```

Comment:
these glyphclass1/glyphclass2 are they supposed to be of type `set` or `tuple`? The docstring of `addClassPair` method says "set"

Reply:
If I use set, I get an error down the line that it it is unhashable type. They are used as a dict key.